docs/core/linux-prerequisites.md

```diff
@@ -91,9 +106,10 @@ The installer bash script is used in automation scenarios and non-admin installa
 ## Install .NET Core dependencies for Red Hat Enterprise Linux (RHEL) 7 Server
 
 > [!Warning]
```

Comment:
it might be easy to miss this. wondering if you should add this as the first step instead.